src/main/java/duke/Message.java

```diff
@@ -0,0 +1,35 @@
+package duke;
+public class Message {
```

Comment:
I really like the fact that you made a Message class! Wish I did this in mine because it would be easier to read many of my methods :)